diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index b8aa711..b084a4b 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -3085,6 +3085,8 @@ pref("browser.mailto.dualPrompt.onLocationChange", false);
 pref("browser.mailto.dualPrompt.dismissNotNowMinutes", 525600); // one year
 pref("browser.mailto.dualPrompt.dismissXClickMinutes", 1440); // one day
 
+#include pulse-browser.js
+
 // Pref to initialize the BackupService soon after startup.
 pref("browser.backup.enabled", true);
 // Pref to control whether scheduled backups run or not.
