diff --git a/browser/base/content/browser.js b/browser/base/content/browser.js
index 0bc3e362057b69754919b0e4a57eeb1846a7e6e5..6cc97a82b354274e8367d6aa00374bf87013a5f6 100644
--- a/browser/base/content/browser.js
+++ b/browser/base/content/browser.js
@@ -1710,6 +1710,7 @@ var gBrowserInit = {
     gPrivateBrowsingUI.init();
     BrowserSearch.init();
     BrowserPageActions.init();
+    VerticalTabs.init();
     if (gToolbarKeyNavEnabled) {
       ToolbarKeyboardNavigator.init();
     }
@@ -10013,6 +10014,7 @@ var FirefoxViewHandler = {
     return document.getElementById(this.BUTTON_ID);
   },
   init() {
+    return;
     CustomizableUI.addListener(this);
 
     ChromeUtils.defineESModuleGetters(this, {
@@ -10021,6 +10023,7 @@ var FirefoxViewHandler = {
     Services.obs.addObserver(this, "firefoxview-notification-dot-update");
   },
   uninit() {
+    return;
     CustomizableUI.removeListener(this);
     Services.obs.removeObserver(this, "firefoxview-notification-dot-update");
   },
