--- a/browser/themes/ThemeVariableMap.sys.mjs
+++ b/browser/themes/ThemeVariableMap.sys.mjs
@@ -159,6 +159,18 @@
       },
     },
   ],
+  [
+    "--in-content-link-color",
+    {
+      lwtProperty: "icon_color",
+    },
+  ],
+  [
+    "--in-content-link-color-hover",
+    {
+      lwtProperty: "icon_color",
+    },
+  ],
 ];
 
 export const ThemeContentPropertyList = [
@@ -170,4 +185,10 @@
   "sidebar_highlight_text",
   "sidebar_text",
   "zap_gradient",
+
+  // Pulse
+  "button_primary",
+  "button_primary_hover",
+  "button_primary_active",
+  "button_primary_color",
 ];

