--- a/browser/themes/ThemeVariableMap.sys.mjs
+++ b/browser/themes/ThemeVariableMap.sys.mjs
@@ -114,6 +114,9 @@
     "--sidebar-background-color",
     {
       lwtProperty: "sidebar",
+      // Pulse: we want to use the sidebar color in the sidebar tabs, which are
+      // under the `browser` parent rather than the `sidebar-box` parent
+      optionalElementID: "browser",
       processColor(rgbaChannels) {
         if (!rgbaChannels) {
           return null;
@@ -159,6 +162,18 @@
       },
     },
   ],
+  [
+    "--in-content-link-color",
+    {
+      lwtProperty: "icon_color",
+    },
+  ],
+  [
+    "--in-content-link-color-hover",
+    {
+      lwtProperty: "icon_color",
+    },
+  ],
 ];
 
 export const ThemeContentPropertyList = [
@@ -170,4 +185,10 @@
   "sidebar_highlight_text",
   "sidebar_text",
   "zap_gradient",
+
+  // Pulse
+  "button_primary",
+  "button_primary_hover",
+  "button_primary_active",
+  "button_primary_color",
 ];

